examples/generate.sh

```diff
@@ -287,4 +293,4 @@ echo "WARNING: ${PROVIDER_COMPONENTS_GENERATED_FILE} includes vSphere credential
 
 # If running in Docker then ensure the contents of the OUT_DIR have the
 # the same owner as the volume mounted to the /out directory.
-[ "${DOCKER_ENABLED-}" ] && chown -R "$(stat -c '%u:%g' /out)" "${OUT_DIR}"
+#[ "${DOCKER_ENABLED-}" ] && chown -R "$(stat -c '%u:%g' /out)" "${OUT_DIR}"
```

Comment:
Hi @figo,

There is a reason this line is here. Why did you comment it out?

Reply:
sorry, thats just for my local testing, will restore it,